### PR TITLE
New version: tencent.tencentmeeting 3.3.2.437

### PR DESCRIPTION
**Pull request by lychichem**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/denelon/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/38859)

### CODE REVIEW

File: manifests/t/Tencent/TencentMeeting/3.3.2.437/Tencent.TencentMeeting.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.locale.1.0.0.schema.json
+
+PackageIdentifier: Tencent.TencentMeeting
+PackageVersion: 3.3.2.437
```
Comment (ItzLevvie) [inline manifests/t/Tencent/TencentMeeting/3.3.2.437/Tencent.TencentMeeting.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 3.2.5.402
```

File: manifests/t/Tencent/TencentMeeting/3.3.2.437/Tencent.TencentMeeting.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: Tencent.TencentMeeting
+PackageVersion: 3.3.2.437
```
Comment (ItzLevvie) [inline manifests/t/Tencent/TencentMeeting/3.3.2.437/Tencent.TencentMeeting.installer.yaml:5]:
```suggestion
PackageVersion: 3.2.5.402
```

File: manifests/t/Tencent/TencentMeeting/3.3.2.437/Tencent.TencentMeeting.locale.zh-CN.yaml [hunk 3]
```diff
@@ -0,0 +1,27 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: Tencent.TencentMeeting
+PackageVersion: 3.3.2.437
```
Comment (ItzLevvie) [inline manifests/t/Tencent/TencentMeeting/3.3.2.437/Tencent.TencentMeeting.locale.zh-CN.yaml:5]:
```suggestion
PackageVersion: 3.2.5.402
```

File: manifests/t/Tencent/TencentMeeting/3.3.2.437/Tencent.TencentMeeting.yaml [hunk 4]
```diff
@@ -0,0 +1,8 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: Tencent.TencentMeeting
+PackageVersion: 3.3.2.437
```
Comment (ItzLevvie) [inline manifests/t/Tencent/TencentMeeting/3.3.2.437/Tencent.TencentMeeting.yaml:5]:
```suggestion
PackageVersion: 3.2.5.402
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
image

File: manifests/t/Tencent/TencentMeeting/3.3.2.437/Tencent.TencentMeeting.yaml (hunk 4 above)
Comment (lychichem) [reply manifests/t/Tencent/TencentMeeting/3.3.2.437/Tencent.TencentMeeting.yaml:5]: I edited files in 3.2.5.402 by mistake and reverted, but I forgot to edit files for 3.3.2.437. Now it's fixed.

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**